content/millennium/r4/clinical/diagnostics/questionnaire-response.md

```diff
@@ -41,6 +41,7 @@ The following fields are returned if valued:
 
 * [Choice Answer]
 * [Note]
+* [Modifiable]
```

Comment:
Is Modifiable

Reply:
https://github.com/cerner/fhir.cerner.com/pull/697/commits/9c5a0a983df9702bbae651e4be4907edbd35b377#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L44